warehouse/templates/manage/account/publishing.html

```diff
@@ -188,6 +188,77 @@
   </form>
 {% endmacro %}
 
+{% macro activestate_form(request, pending_activestate_pubisher_form) %}
+  {{ form_error_anchor(pending_activestate_pubisher_form) }}
+  <form method="POST" action="{{ request.route_path('manage.account.publishing') }}#errors">
+    <input name="csrf_token" type="hidden" value="{{ request.session.get_csrf_token() }}">
+    {{ form_errors(pending_activestate_pubisher_form) }}
+    <div class="form-group">
+      <label for="project_name" class="form-group__label">
+        {% trans %}PyPI Project Name{% endtrans %}
+        {% if pending_activestate_pubisher_form.project_name.flags.required %}
+        <span class="form-group__required">{% trans %}(required){% endtrans %}</span>
+        {% endif %}
+      </label>
+      {{ pending_activestate_pubisher_form.project_name(placeholder=gettext("project name"), autocomplete="off", autocapitalize="off", spellcheck="false", class_="form-group__field", aria_describedby="project_name-errors") }}
```

Comment:
I hate to do this, but I just merged #10003 that uses newlines to make these longer lines more readable. Would you mind reflowing these in the rest of the changed template code?

```suggestion
      {{ pending_activestate_pubisher_form.project_name(
          placeholder=gettext("project name"),
          autocomplete="off",
          autocapitalize="off",
          spellcheck="false",
          class_="form-group__field",
          aria_describedby="project_name-errors"
      ) }}
```

Reply:
MIIIiiiIIiiIIIiIIiiIIiiIIIIIke (pouty face emoji and said with a whiny lilt)

Reply:
Kidding.  I prefer the new format muchly.  I'll do that for sure.